yaas/utils.py

```diff
@@ -0,0 +1,34 @@
+# coding: utf-8
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+def remove_hrefs(el):
```

Comment:
My knee-jerk reaction to this is definitely negative, but maybe I should see it in action?